### PR TITLE
CESNET-MCCG2 got image-sync enabled and relatedegi-ops project

### PR DESCRIPTION
**Pull request by freznicek**
This PR introduces:
 * image-sync enabled
 * ops project created (egi-ops)

### CODE REVIEW

**Review by enolfc — APPROVED** [review]
LGTM

File: sites/CESNET-MCCG2.yaml [hunk 1]
```diff
@@ -1,4 +1,11 @@
 ---
 gocdb: CESNET-MCCG2
 endpoint: https://identity.brno.openstack.cloud.e-infra.cz/v3
-vos: []
+images:
+  sync: true
+  formats:
+    - raw
+vos:
+  - name: egi-ops
```
[the remark this answers is not in the document]
Comment (freznicek) [reply sites/CESNET-MCCG2.yaml:9]: ok, got it.

**Comment by sebastian-luna-valero** [comment]
Adding the site to `fedcloudclient` : https://github.com/tdviet/fedcloudclient/pull/208

File: sites/CESNET-MCCG2.yaml [hunk 2]
```diff
@@ -1,4 +1,11 @@
 ---
 gocdb: CESNET-MCCG2
 endpoint: https://identity.brno.openstack.cloud.e-infra.cz/v3
```
[the remark this answers is not in the document]
Comment (sebastian-luna-valero) [reply sites/CESNET-MCCG2.yaml:3]: For comparison, see the [GOCDB entry](https://goc.egi.eu/gocdbpi/public/?method=get_service_endpoint&sitename=CESNET-MCC&service_type=org.openstack.nova) for `CESNET-MCC`

[the remark this answers is not in the document]
Comment (freznicek) [reply sites/CESNET-MCCG2.yaml:3]: Dear Sebastian,
thank you for pointing it out, I'm going to change it as you suggested but I'm affraid its going wrong direction.

My understanding is following:
* Here in EGI-Federation/fedcloud-catchall-operations are clouds identified via openstack identity Keystone endpoint (although it is not explit, many of clouds use keystone port 5000)
* validator should match endpoint here against GOCDB service endpoint lebeled with org.openstack.keystone ostack service not the org.openstack.nova one.

I agree there is a mismatch between CESNET-MCC and CESNET-MCCG2 clouds, but older CESNET-MCC is wrong not the newer one.

To move forward I changed the org.openstack.nova endpoint to keystone one so validator should be happy now.

--

I believe the problem is here in the validator: https://github.com/EGI-Federation/fedcloud-catchall-operations/blob/664152adb6c539112f1d7cade2cc7fd450ff85a0/validate.sh#L25-L26

I'd expect that validator should match EGI-Federation/fedcloud-catchall-operations site endpoint against org.openstack.keystone service URL in GOCDB and if it is not present then could fallback to org.openstack.nova one. I believe thich change should not break the pipeline.

If that is something you all agree I can push suggested changes to pipeline validator.

[the remark this answers is not in the document]
Comment (enolfc) [reply sites/CESNET-MCCG2.yaml:3]: Due to historic reasons, the GOC entries that we care about are those with `service_type` equal to `org.openstack.nova`, but we are not expecting a nova endpoint but a keystone endpoint to be declared in the URL.

Basically when we started building the fedcloud, nova was our only target, although soon we discovered that the nova URL would be useless without the keystone URL. So we did include that URL as part of the `org.openstack.nova` entry in GOC. Indeed, not what one would expect, but we haven't bothered to change this so far. This is matching the [documentation](https://docs.egi.eu/providers/cloud-compute/registration/#expected-services) as well. 

We may want to change that, but it's not something that we can just do without a transition phase. Every component dealing with the infra will search for the nova entry in GOC and not the keystone one, so we would need to probably live with both for a certain period of time and then migrate every tool one by one. 

So for now, it's `org.openstack.nova` with the keystone URL.

**Review by sebastian-luna-valero — APPROVED** [review]
Many thanks!

**Review by enolfc — APPROVED** [review]
LGTM! Thanks

**Pull request merged**